pkg/kubelet/kubelet.go

```diff
@@ -2044,6 +2123,11 @@ func (kl *Kubelet) ResyncInterval() time.Duration {
 
 // ListenAndServe runs the kubelet HTTP server.
 func (kl *Kubelet) ListenAndServe(address net.IP, port uint, tlsOptions *server.TLSOptions, auth server.AuthInterface, enableDebuggingHandlers, enableContentionProfiling bool) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletServerCertificate) && tlsOptions != nil {
+		tlsOptions.Config.GetCertificate = func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
+			return kl.serverCertificateManager.Current(), nil
```

Comment:
If there's no bootstrap cert, this could be nil right? Should that be an error case, or is it ok to just return `nil, nil` here?